File: 30 Day Code Challenges/Day 06 Review/Solution.java
import java.io.*;
import java.util.*;

public class Solution {

    static String oddEvenParser(char[] arr) {
        String even = "";
        String odd = "";
        for (int i = 0; i < arr.length; i++) {
            if (i % 2 == 0 || i == 0) {
                even += arr[i];
            } else {
                odd += arr[i];
            }
        }
        return even + " " + odd;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = Integer.parseInt(sc.nextLine());
        for (int i = 0; i < t; i++) {
            System.out.println(oddEvenParser(sc.nextLine().toCharArray()));
        }

        sc.close();
    }
}

File: 30 Day Code Challenges/Day 06 Review/dotnet/Solution.cs
using System;
using System.Collections.Generic;
using System.IO;

namespace dotnet
{
    class Solution
    {
        static String oddEvenParser(String str) {
            String even = "";
            String odd = "";
            for (int i = 0; i < str.Length; i++) {
                if (i % 2 == 0) {
                    even += str[i];
                } else {
                    odd += str[i];
                }
            }
            return even + " " + odd;
        }
        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());
            for (int i = 0; i < n; i++) {
                Console.WriteLine(oddEvenParser(Console.ReadLine()));
            }
        }
    }
}
